components/alignmentannotation/AlignmentAnnotationTable-metadata.json

```diff
@@ -0,0 +1,45 @@
+{
+    "url": "alignmentAnnotation.csv",
+    "dc:conformsTo": "http://cldf.clld.org/v1.0/terms.rdf#alignmentAnnotationTable",
+    "tableSchema": {
+        "columns": [
+            {
+                "name": "ID",
+                "required": true,
+                "propertyUrl": "http://cldf.clld.org/v1.0/terms.rdf#id",
+                "datatype": {
+                    "base": "string",
+                    "format": "[a-zA-Z0-9_\\-]+"
+                }
+            },
+            {
+                "name": "Alignment_ID",
+                "required": true,
+                "propertyUrl": "http://cldf.clld.org/v1.0/terms.rdf#alignmentReference",
+                "datatype": "string"
+            },
+            {
+                "name": "Segment_Slice",
+                "required": false,
+                "propertyUrl": "http://cldf.clld.org/v1.0/terms.rdf#segmentSlice",
+                "datatype": {
+                    "base": "string",
+                    "format": "\\d+(:\\d+)?"
+                },
+                "separator": " "
+            },
+            {
+                "name": "AnnotationKind",
```

Comment:
I think the word "kind" is not right, at least it sounds extremely weird to me as people don't use this often in compounds for terms, I'd prefer "type", why not use "annotation type"?

Reply:
fine!